<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseWidgets::Internal::AustrianSocialNumberEdit</class>
 <widget class="QWidget" name="BaseWidgets::Internal::AustrianSocialNumberEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="leSVNR">
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leBirthDate">
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labelNumberOk">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leSVNR</tabstop>
  <tabstop>leBirthDate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
